src/pytest_docker/plugin.py

```diff
@@ -119,6 +142,27 @@ def wait_until_responsive(
 
         raise Exception("Timeout reached while waiting on service!")
 
+    def display_live_logs(self, service: str) -> None:
+        if service in self._live_logs:
+            return
+        self._live_logs[service] = self._thread_pool_executor.submit(
+            self._docker_compose.execute, f"logs {service} -f"
+        )
+
+    def close(self) -> None:
+        for _, fut in self._live_logs.items():
+            _ = fut.cancel()
+        self._thread_pool_executor.shutdown(wait=False, cancel_futures=True)
```

Comment:
`cancel_futures` was added in 3.9, we're aiming for  3.8 compatibility - and you're cancelling them yourself anyway.

Reply:
removed